### ISSUE TITLE
Samogitian (sgs)

### ISSUE THREAD

**Issue by Guybrush88** [Tatoeba/tatoeba2#1178]
- Language name: Samogitian
- Language info: https://en.wikipedia.org/wiki/Samogitian_dialect
- ISO 639-3 code: sgs
- Public List: https://tatoeba.org/ita/sentences_lists/show/6242
- Icon: https://en.wikipedia.org/wiki/Samogitia#/media/File:Flag_of_Samogitia.png
- Request by: https://tatoeba.org/ita/user/profile/Kova99

**Comment by RyckRichards** [comment]
[for dev.tatoeba.org]
CALL add_new_language('sgs', 4253);

[for tatoeba.org]
CALL add_new_language('afh', 6242);

http://dev.tatoeba.org/eng/sentences_lists/show/4253

**Comment by sabretou** [comment]
Created a 30x20 version of the flag with transparency along the sides and between the tails, just in case it is needed: https://drive.google.com/open?id=0B5_jtoG7OkkpR2dxdmlnT21NcE0

**Comment by RyckRichards** [comment]
Thank you so really much, sabretou!

**Comment by ckjpn** [comment]
I wonder if since the shape of this flag it the way it is, perhaps no border is better.

Here's an example, re-shaped a bit to fully use the 30x20 pixels.

![sgs-full-30x20](https://cloud.githubusercontent.com/assets/7934273/15763985/4e7e6e2e-2962-11e6-8e82-30c131f3e7d5.png)

Her's an example with only the crest.  I just did this to see if it might be better.  Hoever, I think the other looks better, though.  The shape of the flag is unique, so it will be easy to recognize.

![sgs-part-30x20](https://cloud.githubusercontent.com/assets/7934273/15763971/484e2b48-2962-11e6-8a4c-adf99050814a.png)

**Comment by cueyayotl** [comment]
Here's my takes on the language icon:
http://imgur.com/Vu9tKMh
I used the original dimensions, but unlike sabretou I didn't use indexed color mode when shrinking the image, so the right part of the image looks a bit smoother.
OR
http://imgur.com/JR7YaE2
Following CK's second example, I made an icon based ONLY on the crest and NOT the background.

I personally like my first one better as do I like CK's first example above with the flag resized.

It may be better to keep the original proportion of the flag, but I am OK with either my or CK's example above.
